ProcessOut/Classes/ProcessOut.swift

```diff
@@ -420,7 +443,7 @@ public class ProcessOut {
     ///   - handler: 3DS2 handler
     ///   - with: UIViewController to display webviews and perform fingerprinting
     public static func makeCardToken(source: String, customerId: String, tokenId: String, handler: ThreeDSHandler, with: UIViewController) {
-        let tokenRequest = TokenRequest(source: source)
+        let tokenRequest = TokenRequest(source: source, thirdPartySDKVersion: "")
```

Comment:
does this hardcode the sdk version to a blank string?

Reply:
It will yes, but this is the overload that wouldn't cause regression and the one merchants are currently using. Hence why we choose to overload these instead of introducing breaking changes and let the merchant decide if they want to use the other one. Some might not be using 3DS through the mobile SDK as well I would imagine and wouldn't have the need to provide the 3ds version?